Config/SQL/schema_MySQL.sql

```diff
@@ -1,276 +1,277 @@
 -- Schema for PHPRecipeBook 5.0
 CREATE TABLE users (
-	id INT NOT NULL AUTO_INCREMENT,
-	username VARCHAR(32) NOT NULL UNIQUE,
-	password VARCHAR(255) NOT NULL DEFAULT '',
-	name VARCHAR(64) NOT NULL DEFAULT '',
-	access_level INTEGER NOT NULL DEFAULT '0',
-        locked BOOL NOT NULL DEFAULT 0,
-        reset_token VARCHAR(255) NULL,
-        reset_time DATETIME NULL,
-	language VARCHAR(8) DEFAULT 'en' NOT NULL,
-	country VARCHAR(8) DEFAULT 'us' NOT NULL,
-	last_login DATE,
-	email VARCHAR(64) NOT NULL UNIQUE,
-        meal_plan_start_day INT NOT NULL DEFAULT '0', -- Sunday is default
-        created DATETIME,
-        modified DATETIME,
+	`id` INT NOT NULL AUTO_INCREMENT,
+	`username` VARCHAR(32) NOT NULL UNIQUE,
+	`password` VARCHAR(255) NOT NULL DEFAULT '',
+	`name` VARCHAR(64) NOT NULL DEFAULT '',
+	`access_level` INTEGER NOT NULL DEFAULT '0',
+	`locked` TINYINT(1) NOT NULL DEFAULT 0,
+	`reset_token` VARCHAR(255) NULL,
+	`reset_time` DATETIME NULL,
+	`language` VARCHAR(8) DEFAULT 'en' NOT NULL,
+	`country` VARCHAR(8) DEFAULT 'us' NOT NULL,
+	`last_login` DATE,
+	`email` VARCHAR(64) NOT NULL UNIQUE,
+	`meal_plan_start_day` INT NOT NULL DEFAULT '0', -- Sunday is default
+	`created` DATETIME,
+	`modified` DATETIME,
 	PRIMARY KEY (id)
 );
 
 CREATE TABLE stores ( 
-	id INT NOT NULL AUTO_INCREMENT,
-	name VARCHAR(32) NOT NULL DEFAULT '',
-	layout TEXT,
+	`id` INT NOT NULL AUTO_INCREMENT,
+	`name` VARCHAR(32) NOT NULL DEFAULT '',
+	`layout` TEXT,
+	`user_id` INT NULL REFERENCES users(id) ON DELETE SET DEFAULT ON UPDATE CASCADE,
```

Comment:
Stores do not actually have a relationship to users.  The problem is later in the script during the insert to stores.  The insert should be updated to remove the user_id (1) insert.  You can see what the model is really by looking at Config\Schema\schema.php